src/Share/Share.router.ts

```diff
@@ -0,0 +1,63 @@
+import { Response } from 'express'
+import { env } from 'decentraland-commons'
+import { format } from 'util'
+
+import { Router } from '../common/Router'
+
+import { Project } from '../Project/Project.model'
+import { withSocialUserAgentDetector, SocialRequest } from '../middleware/share'
+import { Params } from './Share.types'
+import { Pool, PoolAttributes } from '../Pool'
+import { ProjectAttributes } from '../Project'
+import template from './template'
+
+const BUILDER_URL = env.get('BUILDER_URL', '')
+
+export class ShareRouter extends Router {
+  mount() {
+    /**
+     * Redirect to scene
+     */
+    this.router.get(
+      '/share/:type(scene|pool)/:id',
+      withSocialUserAgentDetector,
+      this.redirectToBuilder
+    )
+  }
+
+  private redirectToBuilder = async (req: SocialRequest, res: Response) => {
+    const { type, id } = req.params as Params
+    const targetPath = type === 'scene' ? `/view/${id}` : `/view/${type}/${id}`
+    const url = BUILDER_URL + targetPath
```

Comment:
This scares me _a bit_. Maybe we should use `url.resolve` to avoid messing up here?